backend/service/authn/oidc.go

```diff
@@ -389,6 +408,10 @@ func NewOIDCProvider(ctx context.Context, config *authnv1.Config, tokenStorage S
 		scopes = defaultScopes
 	}
 
+	if c.GroupsFromUserinfo {
+		scopes = append(scopes, "groups")
+	}
```

Comment:
i don't remember if this is still required with the separate request to userinfo

Reply:
yeah it is